### ISSUE TITLE
use-hopper-check breaks vanilla hoppers

### ISSUE THREAD

**Issue by wellnesscookie** [CobbleSword/NachoSpigot#202]
NachoSpigot did implement paper's patch for hopper optimisation, but didn't remove the `'use-hopper-check'` feature in paper's config, nor code, as it was removed for knowing to cause issues with hopper behaviour.

There is no more reason to have it anymore, after the hopper optimisation patch was applied.
Check out: https://github.com/PaperMC/Paper/issues/197

**Comment by CyberFlameGO** [comment]
I'll try to take a look at removing it later today

**Comment by Elierrr** [comment]
> The problem, wherever it's coming from, is Nacho related. After switching to the latest version of Paper for 1.8.8 and turning off the use-hopper-check feature, this was not the case anymore. Even though we did disable that option on Nacho too, the bug was still there.

Try running it on [TacoSpigot](https://github.com/Elierrr/TacoSpigot/releases/download/1.8.8/TacoSpigot.jar).

**Comment by Lucaskyy** [comment]
Is this still an issue?

**Comment by wellnesscookie** [comment]
Yes, at least on Nacho, but not on Paper. We would really love to test it with TacoSpigot, but our players would really hate to see the bug again, and since it does not occurs instantly and it's not possible for it to test whenever we want, we would have to make them test it. I am sorry, even though we would love to use Nacho/Burrito, we would not like to test it on production again.

We can try on our test server, but I am not sure if we would be able to reproduce it ourselves.

**Comment by Lucaskyy** [comment]
Thanks for the response :)
Since this was removed from Paper anyway, it makes sense for NachoSpigot to remove it too.
So that's what we'll be doing soon, so hopefully, that will solve your issue.
